po/de.po

```diff
@@ -13456,6 +13459,8 @@ msgid ""
 "\n"
 "you should not touch values here !"
 msgstr ""
+"Internes Modul um Sensoreigenschaften zu einzustellen.\n"
```

Comment:
WTF? Thie German sentence is completely unrelated to the English wording.

Reply:
Source:
```
 internal module to setup technical specificities of raw sensor.

you should not touch values here !
```